railties/test/generators/model_generator_test.rb

```diff
@@ -319,6 +319,15 @@ def test_fixture_is_skipped_if_fixture_replacement_is_given
     assert_no_file "test/fixtures/accounts.yml"
   end
 
+  def test_fixture_without_pluralization
+    ActiveRecord::Base.pluralize_table_names = false
+    run_generator
+    assert_generated_fixture("test/fixtures/account.yml",
+                             {"one"=>{"name"=>"MyString", "age"=>1}, "two"=>{"name"=>"MyString", "age"=>1}})
+  ensure
+    ActiveRecord::Base.pluralize_table_names = true
```

Comment:
it's better to use a local variable instead of hardcoding the default value:

```ruby
original_pluralize_table_name = ActiveRecord::Base.pluralize_table_names
ActiveRecord::Base.pluralize_table_names = false

ensure
  ActiveRecord::Base.pluralize_table_names = original_pluralize_table_name
end
```